README.md

```diff
@@ -183,10 +183,50 @@ bin/slider destroy presto1
 bin/slider create presto1 --template appConfig.json --resources resources.json
 ```
 
-### Ambari Integration
+# Ambari Integration
 
-TODO:
+
+You can also deploy Presto in Yarn via Ambari. Ambari provides Slider integration and also supports deploying any Slider application package using Slider 'views'. Slider View for Ambari delivers an integrated experience for deploying and managing Slider apps from Ambari Web.
+
+The steps for deploying Presto on Yarn via Slider views in Ambari are:
+
+* Install Ambari Server:
```

Comment:
I think that people who want to use presto with ambari already have ambari installed. Maybe just a link to to ambari installation process or just mention that user needs ambari installed could be enough.
What do you think?